src/App.tsx

```diff
@@ -1,61 +1,126 @@
+import React, { useEffect, useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { Todo } from './Types/InterfaceTodo';
+
+import { TodoList } from './components/TodoList';
+
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+const preparedTodos = todosFromServer.map(todo => ({
+  ...todo,
+  user: usersFromServer.find(user => user.id === todo.userId),
+}));
 
 export const App = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedUserID, setSelectedUserID] = useState('0');
+  const [title, setTitle] = useState('');
+  const [isTitleError, setIsTitleError] = useState(false);
+  const [isUserError, setIsUserError] = useState(false);
+
+  useEffect(() => {
+    setTodos(preparedTodos);
+  }, []);
+
+  const addedNewTodo = (newTodo: Todo) => {
+    setTodos([...todos, newTodo]);
+  };
+
+  const onTitleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
Avoid using one-letter variable names like `e`. Instead, use more descriptive names such as `event` to improve code readability and maintainability. This aligns with the checklist item: "Don't use one-letter variable naming."